arch/src/main/java/arch/view/BaseDialogFragment.kt

```diff
@@ -77,10 +77,11 @@ abstract class BaseDialogFragment<B : ViewDataBinding, VM : BaseDialogViewModel>
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        super.onCreateView(inflater, container, savedInstanceState)
         binding.root.findViewById<Button>(R.id.button_positive)?.setOnClickListener { dispatchResult(BUTTON_POSITIVE) }
         binding.root.findViewById<Button>(R.id.button_negative)?.setOnClickListener { dispatchResult(BUTTON_NEGATIVE) }
         binding.root.findViewById<Button>(R.id.button_neutral)?.setOnClickListener { dispatchResult(BUTTON_NEUTRAL) }
-        return super.onCreateView(inflater, container, savedInstanceState)
+        return binding.root

```

Comment:
Proč tato změna?

Reply:
Kód se zde odkazuje na binding, ale ten se inicializuje v BaseArchDialogFragment, takže se musí super.onCreateView() zavolat jako první a nakonec vrátit root. Původní implementace mi způsobovala automaticky pád. Ale BaseDialogFragment jsem nakonec stejně nepoužil.

Reply:
Asi bych teda neměnil kód, pokud to v PR nepotřebuješ

Reply:
Vráceno